### PR TITLE
Simplified drill install by using minio VIP.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
This depends on the following universe PR: https://github.com/mesosphere/universe/pull/1338

Also when testing, please consider that the dcos marathon app command will reference the old version (until this PR is commited). Just use the local file (or copy past the AppDefinition in the UI).

File: applogs/1.9/README.md [hunk 1]
```diff
@@ -73,13 +73,10 @@ Note that you can learn more about Minio and the credentials in the respective [
...
 you to interactively explore heterogenous datasets across data sources (CSV, JSON, HDFS, HBase, MongoDB, S3).
 
-A prerequisite for the Drill install that the `$PUBLIC_AGENT_IP` environment variable is set, which should have done in the previous step.
-
 Now do the following to install Drill:
 
 ```bash
-$ cd $DEMO_HOME/applogs/1.9/
-$ ./install-drill.sh
+dcos marathon app add https://raw.githubusercontent.com/dcos/demos/master/applogs/1.9/drill/apache-drill.json
```
Comment (pleia2) [inline applogs/1.9/README.md]: If we want to be consistent with other instructions, we should instead use:

$ cd $DEMO_HOME/applogs/1.9/
$ dcos marathon app add drill/apache-drill.json

**Review by pleia2 — CHANGES_REQUESTED** [review]
One comment inline about consistency.

And when I tested this, Drill gave me an error back:

org.apache.drill.common.exceptions.UserRemoteException: SYSTEM ERROR: UnknownHostException: test.minio.marathon.l4lb.thisdcos.directory: unknown error [Error Id: 9cb51213-f912-4146-ac06-ede011f46a4a on ip-10-0-1-151.us-west-2.compute.internal:31010]

File: applogs/1.9/README.md (hunk 1 above)
Comment (pleia2) [inline applogs/1.9/README.md]: Oops, didn't include the universe patch that the PR requires.

Comment (joerg84) [inline applogs/1.9/README.md]: You need to include the changed universe universe PR...

Comment (joerg84) [inline applogs/1.9/README.md]: I hope to get to a point where you don't need to checkout the repo...
Hence, the marathon app add from github

Comment (pleia2) [inline applogs/1.9/README.md]: Ah, gotcha.

Comment (pleia2) [inline applogs/1.9/README.md]: Error still occurs even once we have the Minio VIP working, @joerg84 investigating.

**Comment by mattj-io** [comment]
testing

**Comment by yankcrime** [comment]
Closing as stale.

**Pull request closed without merging**